packages/fxa-auth-server/lib/error.js

```diff
@@ -95,6 +95,8 @@ const ERRNO = {
   REJECTED_CUSTOMER_UPDATE: 181,
   REFRESH_TOKEN_UNKNOWN: 182,
   INVALID_EXPIRED_OTP_CODE: 183,
+  SUBSCRIPTION_ALREADY_CHANGED: 184,
```

Comment:
Do these new error codes need to be handled by the front end?

Reply:
Yup.